IDEAS/Examples/IBPSA/SingleZoneResidentialHydronicHeatPump.mo

```diff
@@ -21,6 +22,7 @@ model SingleZoneResidentialHydronicHeatPump
     mSenFac=1,
     n50=10,
     bouTypFlo=IDEAS.Buildings.Components.Interfaces.BoundaryType.SlabOnGround,
+    hasInt=true,
```

Comment:
We should see how to best set the value of the parameter `lInt`. This is the internal wall length. By default it equals the length of side A but it should be longer. If we assume that each room within the zone is 4x4 meters then we have 12 rooms. Only the outer walls of those 12 rooms are implemented so we need `lInt = 3 x w + 2 x l` for representing the internal wall mass. @JavierArroyoBastida @dhblum do you agree with this reasoning?

@JavierArroyoBastida if you implement the change, it would be interesting to check how the simulation results change.

Reply:
Sounds reasonable, I'll implement the edit and show the simulation results change here.

Reply:
Sounds reasonable to me.

Reply:
I've increased internal wall mass here: https://github.com/open-ideas/IDEAS/pull/1193/commits/87620802dfe39a607f11bf312d775bc735748cfc when assuming 12 rooms. 
Below you can see the simulation results of the operative temperature, supply water temperature and HP modulating signal. A comparison is made between the original implementation, i.e. original size and without any internal wall (original is simulation 1 and modified is simulation 3).  
![image](https://user-images.githubusercontent.com/22677958/108359661-915fda80-71f0-11eb-8dc3-41ecc7ff3cd4.png).

The effects of reducing size and adding internal walls sort of compensate each other regarding thermal mass, although they also increase supply water temperature to the radiant floor. I've increased the offset used in control (in here https://github.com/open-ideas/IDEAS/pull/1193/commits/c14701161679ebd5f6a8c06330395a527d9c34f7) to avoid higher supply water temperatures. See below simulation results (simulation 3) again against the very original implementation (simulation 2). 

![image](https://user-images.githubusercontent.com/22677958/108360179-2a8ef100-71f1-11eb-8a6e-c758ece950bd.png)